lib/ansible/modules/utilities/logic/wait_for_connection.py

```diff
@@ -89,11 +89,12 @@
       customization:
         hostname: '{{ vm_shortname }}'
         runonce:
-        - powershell.exe -ExecutionPolicy Unrestricted -File C:\Windows\Temp\ConfigureRemotingForAnsible.ps1 -CertValidityDays 3650 -ForceNewSSLCert
+        - powershell.exe -ExecutionPolicy Unrestricted -File C:\Windows\Temp\ConfigureRemotingForAnsible.ps1 -CertValidityDays 3650 -ForceNewSSLCert -EnableCredSSP
```

Comment:
Unfortunately adding '-EnableCredSSP' has angered the PEP8(ish) line length check..

```
ERROR: Found 1 pep8 issue(s) which need to be resolved:
2017-03-08 12:27:41 ERROR: lib/ansible/modules/utilities/logic/wait_for_connection.py:92:161: E501 line too long (163 > 160 characters)
```